src/modules/Elsa.ProtoActor/Features/ProtoActorFeature.cs

```diff
@@ -85,14 +78,17 @@ public override void Apply()
         // Register ActorSystem.
         services.AddSingleton(sp =>
         {
-            var systemConfig = Proto.ActorSystemConfig
+            var actorSystemConfig = ActorSystemConfig
                 .Setup()
-                .WithDiagnosticsLogLevel(LogLevel.Debug)
+                .WithDiagnosticsLogLevel(DiagnosticsLogLevel)
                 .WithMetrics();
 
-            var clusterProvider = ClusterProvider(sp);
-            var system = new ActorSystem(systemConfig).WithServiceProvider(sp);
-            var workflowGrainProps = system.DI().PropsFor<WorkflowInstanceActor>();
+            ConfigureActorSystemConfig(sp, actorSystemConfig);
+
+            var clusterProvider = CreateClusterProvider(sp);
+            var system = new ActorSystem(actorSystemConfig).WithServiceProvider(sp);
+
+            var workflowGrainProps = system.DI().PropsFor<WorkflowInstanceActor>().WithTracing();
```

Comment:
Do we always want tracing enabled, or make this configurable?

Reply:
Fair question - not everyone may be interested in the overhead of sending OpenTelemetry signals. I'll make it configurable.